pkg/v1/tkg/client/delete_region.go

```diff
@@ -152,7 +152,8 @@ func (c *TkgClient) DeleteRegion(options DeleteRegionOptions) error { //nolint:f
 
 		// If clusterclass feature flag is enabled then deploy management components
 		if config.IsFeatureActivated(config.FeatureFlagPackageBasedLCM) {
-			if err = c.InstallOrUpgradeManagementComponents(cleanupClusterKubeconfigPath, "", false); err != nil {
+			if err = c.InstallManagementComponentsInCleanupCluster(cleanupClusterKubeconfigPath,
```

Comment:
I think we have 2 options to avoid change to many code:

1. Retrieve all key/values from secret tkg-pkg-tkg-system-values and set them to `c.TKGConfigReaderWriter()` before we call `c.InstallOrUpgradeManagementComponents`.
2. Add a parameter `tkgPackageValuesFile` to `c.InstallOrUpgradeManagementComponents` allowing overwrite `tkgpackagevalues.yaml`

Reply:
Thank you @DanielXiao  for the review, I will refactor the commit per our discussion offline.